style/ruby/README.md

```diff
@@ -38,6 +38,9 @@ Ruby
 * Use heredocs for multi-line strings.
 * Prefer `protected` over `private` for non-public `attr_reader`s, `attr_writer`s,
   and `attr_accessor`s.
+* Break up method definitions over 80 characters long by
```

Comment:
yeah sorry that was not the rule I meant to highlight (copy-pasted too quickly). These are the related rules I was thinking of :

`Break long lines after 80 characters.`
 `If you break up an argument list, keep the arguments on their own lines and closing parenthesis on its own line.` 

from https://github.com/thoughtbot/guides/blob/803a13409e72ceb7e42c6a887922e359a0fd8da5/style/README.md